### DISCUSSION TITLE
Export to Neo4J

### DISCUSSION THREAD

**Discussion by pierrevanrensburg** [chanhx/crabviz#43] [Ideas]
It would be great to export a call graph to Neo4J database.  You can view and further process it from there using Neo4J GUI, run graphing algorithms, and more.  This could relieve all the issues with viewing call graphs and provide focus to this project on the core functionality of reading code (across various languages) and producing call graphs. Any thoughts?

**Comment by chanhx** [comment]
I had considered using an in-memory graph database internally to store the data and do some analysis, but hadn't thought about exporting call graphs to a graph database. Thanks for the suggestion, it's a great feature and very helpful for code analysis. But actually I don't know much about graph databases, any advice on what format I should export to be compatible with as many graph databases as possible?
